test/sumOfOther.test.js

```diff
@@ -0,0 +1,27 @@
+const assert = require('assert');
+const sumOfOther = require('../sumOfOther');
+
+describe('Sum of elements excluding current', function() {
+      it('should return the array with sum of other elements on each index 1', function() {
```

Comment:
All tests should have appropriate description instead of using sequence numbers. In addition you should test not only positive cases, but also negative.